keyboards/takashicompany/goat51/info.json

```
@@ -0,0 +1,490 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "GOAT51",
+    "maintainer": "takashicompany",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "D4",
+            "C6",
+            "D7",
+            "E6",
+            "B4",
+            "B5",
+            "D2"
+        ],
+        "rows": [
+            "F4",
+            "F5",
+            "F6",
+            "F7",
+            "B1",
+            "B3",
+            "B2",
+            "B6"
+        ]
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "D1", "pin_b": "D0" }
+         ]
+    },
+    "processor": "atmega32u4",
+    "url": "https://github.com/takashicompany/goat51",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0029",
+        "vid": "0x7463"
+    },
+    "rgblight": {
+        "led_count": 11,
+        "pin": "D3",
+        "hue_steps": 10,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
+        "animations": {
+            "all": true
```

could you explicitly define all of the modes? 

```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```